test/DotNetEnv.Tests/ParserTests.cs

```diff
@@ -361,6 +392,23 @@ public void ParseAssignment ()
             Assert.Throws<ParseException>(() => Parsers.Assignment.End().Parse("EV_DNE='"));
             Assert.Throws<ParseException>(() => Parsers.Assignment.End().Parse("EV_DNE=0\n1"));
 
+            testParse("EV_DNE", "", "EV_DNE=");
+            testParse("EV_DNE", "a b c", "EV_DNE=a b c # comment");
+            testParse("EV_DNE", "test", "EV_DNE= test  #a'bc allow singleQuotes in comment");
+            testParse("EV_DNE", "test", "EV_DNE= test  #a\"bc allow doubleQuotes in comment");
+            testParse("EV_DNE", "test", "EV_DNE= test #a$bc allow dollarSign in comment");
+
+            testParse("EV_DNE", "http://www.google.com/#anchor", "EV_DNE=http://www.google.com/#anchor");
+            testParse("EV_DNE", "", "EV_DNE= #no value just comment");
```

Comment:
Notice this addition, which was critical for me supporting the move to making the next line pass.